.travis.yml

```diff
@@ -20,8 +20,8 @@ before_install:
 
 install:
 - pip install ansible-lint
+- ansible-galaxy collection install operator_sdk.util
```

Comment:
I install the operator_sdk.util collection here because I was having issues with setting paths or whatever `ansible-lint` was expecting for the vendored collection...